getnacelle/nacelle-js
# chore: create intake forms for bugs & enhancements

### WHY are these changes introduced?

Addresses [LS-1348](https://nacelle.atlassian.net/browse/LS-1348)

> Create issue template for nacelle-js



### WHAT is this pull request doing?

Adds two [GitHub Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms):

1. For [**Bug Reports**](https://github.com/getnacelle/nacelle-js/blob/LS-1348-issue-forms/.github/ISSUE_TEMPLATE/bug_report.yml)
2. For [**Feature Requests**](https://github.com/getnacelle/nacelle-js/blob/LS-1348-issue-forms/.github/ISSUE_TEMPLATE/feature_request.yml)

This PR also pares down our contribution docs, as the bug report and feature enhancement guidelines are taken care of in the Issue Forms.

## 

### How to Test

Check out the links in **1** and **2** above. I don't think there's a way to take these forms for a dry run before this is merged.

Note that the form metadata visible in the links above isn't visible to users when submitting a new issue with the form:

<img width="450" alt="GitHub Issue Form metadata" src="https://user-images.githubusercontent.com/5732000/147797421-3a373a18-d0b4-47e7-bfe1-98ec03f3ee26.png">

### Testing checklist


### Next Steps

It might be wise to create a GitHub Action that takes care of adding new packages / examples / starters to the `project` section of the Issue forms. In the meantime, it's not a big deal to maintain this list by hand (it's not changing quickly yet).
